Source/Particles/ParticleBoundaryBuffer.cpp

```diff
@@ -151,7 +152,7 @@ struct CopyAndTimestamp {
             dst.m_runtime_idata[j][dst_i] = src.m_runtime_idata[j][src_i];
         }
         dst.m_runtime_idata[m_step_index][dst_i] = m_step;
-        dst.m_runtime_rdata[m_time_index][dst_i] = m_step*m_dt;
+        dst.m_runtime_rdata[m_delta_index][dst_i] = 0._rt; //delta_fraction is initialized to zero
```

Comment:
Could you open an issue (https://github.com/ECP-WarpX/WarpX/issues) so that we remember to fix this?
(You could also add that we need to save the normal in this case.)

Reply:
https://github.com/ECP-WarpX/WarpX/issues/4718